lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -120,12 +94,15 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
   render(category, groups) {
     const element = this.dom.createElement('div', 'lh-category');
     this.createPermalinkSpan(element, category.id);
-    element.appendChild(this.renderCategoryScore(category));
+    element.appendChild(this.renderCategoryHeader(category));
 
+    // Metrics
     const metricAudits = category.audits.filter(audit => audit.group === 'metrics');
     const metricAuditsEl = this.renderAuditGroup(groups['metrics'], {expandable: false});
+    const estValuesEl = this.dom.createChildOf(metricAuditsEl, 'div',
+    'lh-metrics__disclaimer lh-metrics__disclaimer--head');
+    estValuesEl.textContent = 'Values are estimated and may vary. ';
```

Comment:
aw I liked `Estimated values may vary`, I guess it doesn't necessarily convey that all the values you see are estimated though 🤔 

is the space at the end necessary? :)